File: backend/Models/CartItem.cs
namespace OrtegaTourism.Models
{
    public class CartItem
    {
        public string Id { get; set; } = string.Empty;
        public string Type { get; set; } = string.Empty; // "place", "hotel", "restaurant", "transport"
        public int ItemId { get; set; }
        public string Name { get; set; } = string.Empty;
        public string Description { get; set; } = string.Empty;
        public string ImageUrl { get; set; } = string.Empty;
        public decimal Price { get; set; }
        public string Category { get; set; } = string.Empty;
        public DateTime AddedAt { get; set; }
        public Dictionary<string, object> AdditionalInfo { get; set; } = new Dictionary<string, object>();
    }
}

File: backend-laravel/app/Models/CartItem.php
<?php

namespace App\Models;

class CartItem
{
    public string $id = '';
    public string $type = '';
    public int $itemId = 0;
    public string $name = '';
    public string $description = '';
    public string $imageUrl = '';
    public float $price = 0;
    public string $category = '';
    public string $addedAt = '';
    public array $additionalInfo = [];
    
    public function toArray(): array
    {
        return [
            'id' => $this->id,
            'type' => $this->type,
            'itemId' => $this->itemId,
            'name' => $this->name,
            'description' => $this->description,
            'imageUrl' => $this->imageUrl,
            'price' => $this->price,
            'category' => $this->category,
            'addedAt' => $this->addedAt,
            'additionalInfo' => $this->additionalInfo
        ];
    }
}
